content/en/agent/fleet_automation/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Fleet Automation
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "/agent/remote_config"
+  tag: "Documentation"
+  text: "Find out more about Remote Configuration"
+- link: "/infrastructure/list/#agent-configuration"
+  tag: "Documentation"
+  text: "Learn about the Agent configuration view"
+---
+
+{{< callout btn_hidden="true">}}Fleet Automation is in private beta.{{< /callout >}}
```

Comment:
maybe use the shortcode option to change the title? "Join the Beta!" looks funny when there's no way to actually do so on the page.

Reply:
Yeah, I agree. I'd turned the title off previously, but it looked weird with so little text in the box. I'll think about this some more